socorro/mozilla_settings.py

```diff
@@ -181,7 +181,11 @@ def _or_none(val):
             default="^socorro[0-9]{6}$",
             doc="Regex for matching Elasticsearch index names.",
         ),
-        "url": _config("ELASTICSEARCH_URL", doc="Elasticsearch url."),
+        "url": _config(
+            "LEGACY_ELASTICSEARCH_URL",
+            alternate_keys=["ELASTICSEARCH_URL"],
```

Comment:
if i understand correctly, this will use LEGACY_ELASTICSEARCH_URL if present and fall back to ELASTICSEARCH_URL if not, which should retain backwards compatibility for configuration (for now)